### ISSUE TITLE
Spec `info.json` watcher does not work when copy files in Webstorm

### ISSUE THREAD

**Issue by operatino** [sourcejs/Source#126]
Using Webstorm IDE to move and copy `info.json` files, SourceJS Spec watcher does not trigger `chage` event, and navigation tree is not updated.

Also, watcher should be moved in separate process, so it won't break main app on error, but re-run instead.

**Comment by operatino** [comment]
@cheshirsky, this is one of the bugs I mentioned in our Skype talk.

**Comment by operatino** [comment]
I re-checked again, currently it's working. Only thing that is confusing, is that to see new spec in list, you need to first add spec file like `readme.md` or `index.src.html` and only after `info.json` file.

**Comment by operatino** [comment]
Another use case - when user renames the folder, watcher does not get any events.

**Comment by operatino** [comment]
I solved problem with the order of Spec files creation, and created separate task about folder name change.
